### PR TITLE
Breadcrumbs and translations

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fixes @sarahdib feedback about missing breadcrumbs and translations in #77 
| Type?         | bug fix
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | /
| How to test?  | Breadcrumbs are now displayed and some of the strings now use the core translations.

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

File: views/templates/pages/products-list.tpl [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
-    data-add-to-cart="{l s='Add to cart' d='Modules.Blockwishlist.Shop'}"
+    data-add-to-cart="{l s='Add to cart' d='Shop.Theme.Actions'}"
     data-share="{if $isGuest}true{else}false{/if}"
     data-customize-text="{l s='Customize' d='Modules.Blockwishlist.Shop'}"
-    data-quantity-text="{l s='Quantity' d='Modules.Blockwishlist.Shop'}"
+    data-quantity-text="{l s='Quantity' d='Shop.Theme.Catalog'}"
     data-title="{$wishlistName}"
     data-no-products-message="{l s='No products found' d='Modules.Blockwishlist.Shop'}"
-    data-filter="{l s='Filter' d='Modules.Blockwishlist.Shop'}"
+    data-filter="{l s='Shop.Theme.Actions' d='Modules.Blockwishlist.Shop'}"
```
Comment (atomiix) [inline views/templates/pages/products-list.tpl:36]:
```suggestion
    data-filter="{l s='Filter' d='Shop.Theme.Actions'}"
```

Comment (Quetzacoalt91) [reply views/templates/pages/products-list.tpl]: Woups, thanks!

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by NeOMakinG** [comment]
QA will be done on new-module

**Comment by liviu-v** [comment]
Any one please help me, I don't know how github works, but for the block module wishlist, I can not find translation, for eg it is write "You need to be logged in to save products in your wishlist." where can I translate this part for front end??!

**Comment by matks** [comment]
Hi @liviu-v If  we use GitHub issues only to discuss about bugs and new features for the PrestaShop project. We do not provide help on GitHub.

I would suggest looking for help on
- community Slack channel https://github.com/PrestaShop/open-source/blob/master/slack/readme.md
- community forum https://www.prestashop.com/forums/

See https://devdocs.prestashop.com/1.7/faq/i-need-help/

**Pull request merged**